ecommerce/models_test.py

```diff
@@ -283,11 +283,13 @@ def test_product_managers():
     """Test that the default manager returns only active products and the 'all_objects' manager returns all the
     products
     """
-    in_active_products = ProductFactory.create_batch(3, is_active=False)
+    inactive_products = ProductFactory.create_batch(3, is_active=False)
     active_products = ProductFactory.create_batch(2, is_active=True)
 
-    assert list(Product.objects.all()) == active_products
-    assert list(Product.all_objects.all()) == in_active_products + active_products
+    assert set(Product.objects.all().order_by("id")) == set(active_products)
+    assert set(Product.all_objects.all().order_by("id")) == set(
+        inactive_products + active_products
+    )
```

Comment:
This works but I think if we are ordering the `Product.objects.all().order_by("id")` then the list makes sense. Take it or leave it
```
assert list(Product.objects.all().order_by("id")) == list(active_products)
assert list(Product.all_objects.all().order_by("id")) == list(
      inactive_products + active_products
 )
```